tests/conftest.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+import json
+import uuid
+import os
+from hydra_python_core import doc_maker
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from hydrus.app_factory import app_factory
+from hydrus.data import crud, doc_parse
+from hydrus.data.db_models import Base, create_database_tables
+from hydrus.utils import (set_api_name, set_doc, set_page_size, set_session)
+from hydrus.tests.conftest import get_doc_classes_and_properties, gen_dummy_object
+from hydrus.socketio_factory import create_socket
```

Comment:
Please update hydrus. As the directory structure has been changed recently and some of these imports won't work now.

Reply:
I checked the latest changed structure is not released yet.
The latest released version is  `hydrus==0.4.3`

I will update it using setup.py.